src/Test/TestGameEvent.java

```diff
@@ -182,6 +182,7 @@ boolean testMoveEvent()
     Unit apc = addUnit(testMap, testCo1, UnitEnum.APC, 3, 2);
 
     Path path = new Path(1.0); // TODO: Why do we have to provide a speed here?
+    path.addWaypoint(3, 3); // we need two waypoints to not break compatibility with MoveEvent, since it assumes the first waypoint isn't used.
```

Comment:
I would rephrase this to say that Paths should be defined to start where the Unit currently is (really, we should specify each intermediate point as well, so that fuel cost can be calculated correctly. Speaking of which, it would be good to have a test case to verify we are calculating fuel cost correctly).

Reply:
We're not checking fuel consumption, though, just trying to test if the unit actually moves when the event tells it to.
I could attempt to make the move event or fuel consumption check account for malformed paths, but that would likely break compatibility further.
I don't think a malformed path is a problem here, so accounting for the breaking change seemed like the most logical step.

Reply:
This test wasn't made for fuel consumption/path formation, so I'm fine with leaving this as it is, but it wouldn't hurt to have another test to verify that we consume fuel correctly. I'm not too worried about the path formation thing at this point; we may build a test for that when we fix the GameAction issue we have.